.editorconfig

```diff
@@ -0,0 +1,4 @@
+root = true
```

Comment:
Are these configs added intentionally?

Reply:
Yes, I have seen that the lint checks for the E501 in `/tests/style.py`. It fails without properly telling where it fails, so this config will let the dev knows when he/she crosses the line limit in his/her editor of choice.